src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -233,14 +233,18 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
   const promoImage = promoImageRawBlock?.model?.locator;
 
   const showTopics = Boolean(showRelatedTopics && topics.length > 0);
-
+  const authors = bylineLinkedData
+    .map(data => {
+      return data?.authorName;
+    })
+    .join();
```

Comment:
You don't need to join if you've already mapped, unless you're joining by something like a comma?
```suggestion
```

Reply:
yeah this is so it's a comma seperated list that chartbeat expects, instead of an array of strings

Reply:
Ahh OK. If you don't provide a parameter to join, does it default to a comma? Is it better to be explicit? 
```suggestion
    .join(',');
```

Reply:
Yeah it defaults to a comma - can be explicit if you like! maybe better for readability?

Reply:
Yes, if that's OK?